### PR TITLE
Fix ConcatWithoutSpacesFixer's concat near number

### PR DESCRIPTION
**Pull request by ghost**
check whether or not the token before and after the concat symbol '.' is a number before removing the whitespace.

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/ConcatWithoutSpacesFixer.php [hunk 1]
```diff
@@ -29,8 +29,12 @@ public function fix(\SplFileInfo $file, $content)
 
         foreach ($tokens as $index => $token) {
             if ($token->equals('.')) {
-                $tokens->removeLeadingWhitespace($index, $whitespaces);
-                $tokens->removeTrailingWhitespace($index, $whitespaces);
+                if (!$tokens[$tokens->getPrevNonWhitespace($index)]->isGivenKind(T_LNUMBER)) {
+                    $tokens->removeLeadingWhitespace($index, $whitespaces);
+                }
+                if (!$tokens[$tokens->getNextNonWhitespace($index)]->isGivenKind(T_LNUMBER)) {
```
[the remark this answers is not in the document]
Comment (glensc) [reply Symfony/CS/Fixer/Symfony/ConcatWithoutSpacesFixer.php]: add such fixer then :dancer:

[the remark this answers is not in the document]
Comment (SpacePossum) [reply Symfony/CS/Fixer/Symfony/ConcatWithoutSpacesFixer.php]: >add such fixer then :dancer:

there is such a fixer

[the remark this answers is not in the document]
Comment (SpacePossum) [reply Symfony/CS/Fixer/Symfony/ConcatWithoutSpacesFixer.php]: >add such fixer then :dancer:

there is a fixer for adding a line after the control structure

[the remark this answers is not in the document]
Comment (keradus) [reply Symfony/CS/Fixer/Symfony/ConcatWithoutSpacesFixer.php]: really, touching PR from '14 ? :D

**Pull request closed without merging**